CHANGELOG.md

```diff
@@ -2,6 +2,278 @@
+- Allow class search to match text anywhere in the string by @LianaHarris360 in https://github.com/learningequality/kolibri/pull/13773
+- Remove unused modal by @AlexVelezLl in https://github.com/learningequality/kolibri/pull/13788
+- Add empty messages on classes lists by @AlexVelezLl in https://github.com/learningequality/kolibri/pull/13785
+- [Bulk User]: Handle bad-data errors w/ `handleApiError` (+ better side panel refresh handling) by @nucleogenesis in https://github.com/learningequality/kolibri/pull/13769
+- Filter out soft deleted users from lessons, quizzes and groups by @AlexVelezLl in https://github.com/learningequality/kolibri/pull/13806
+- Prototyping User Table Design Updates by @nucleogenesis in https://github.com/learningequality/kolibri/pull/13808
+- [Copy a class]: Use vuex getter for facility-aware redirection by @nucleogenesis in https://github.com/learningequality/kolibri/pull/13858
+- [BUM Side Panel]: Fix hidden android side panels by @nucleogenesis in https://github.com/learningequality/kolibri/pull/13855
+- have copy learners in copy class workflow set to true by default by @marcellamaki in https://github.com/learningequality/kolibri/pull/13864
+- Refactor UsersTableToolbar by @AlexVelezLl in https://github.com/learningequality/kolibri/pull/13862
+- Updates empty filter state messaging by @AllanOXDi in https://github.com/learningequality/kolibri/pull/13762
+- Manually cherry-picked rebasing of @AllanOXDi 's `filtersupdate` branch #13830 by @nucleogenesis in https://github.com/learningequality/kolibri/pull/13860
+- Fixes Bulk User Creation Failed prop console error by @AllanOXDi in https://github.com/learningequality/kolibri/pull/13830
+- Fix CopyClassModal handle removed coaches by @nucleogenesis in https://github.com/learningequality/kolibri/pull/13879
+- Robustly clear sessions on user deletion to cause immediate logout by @rtibbles in https://github.com/learningequality/kolibri/pull/13757
+
+</details>
+
+
+Noteworthy technical updates include upgrading to Node.js v20, upgrading to Morango for improved sync reliability, implementing SQLite transaction improvements for better performance, and adding Django-silk support for development performance profiling.
+
+Other technical additions include upgrading Morango for improved sync performance, implementing load testing framework for learner interactions, and finalizing JS package renaming and migration in the packages directory.
+
+<details>
+  <summary>List of supporting PRs</summary>
+
+- Morango upgrade and sync queue tweaks by @rtibbles in https://github.com/learningequality/kolibri/pull/13774
+- Add load testing framework for learner lesson interactions by @rtibbles in https://github.com/learningequality/kolibri/pull/13856
+- Use BEGIN IMMEDIATE for all SQLite transactions by default by @rtibbles in https://github.com/learningequality/kolibri/pull/13878
+
+</details>
+
+### Changed
+
+- **BREAKING CHANGE** - Node.js has been upgraded from v18 to v20.
```

Comment:
Again, not sure this is a breaking change for Kolibri per se, only for its published npm packages.